week-4/Homework/mandatory/4-groceries.js

```diff
@@ -3,10 +3,15 @@
 // 2. Loop through the groceryList object to gather the item properties into the groceriesToBuy array.
 // 3. Then use console.log() to print out the list. It should print ['Potatoes', 'Orange Juice', 'Rice']
 
-let groceriesToBuy = [];
+let groceriesToBuy=[]
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange Juice",
+  item3: "Rice"
 };
+for (const property in groceryList){
```

Comment:
Good use of a `for` loop. Perhaps you could have used the Object methods we learned last week to update the values in `groceryList`?